TicketManagement/src/main/java/com/example/TicketManagement/model/Event.java

```diff
@@ -0,0 +1,42 @@
+
+package com.example.TicketManagement.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+
+@Entity
+@Setter
+@Getter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="event")
+public class Event {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Integer eventID;
+
+    @ManyToOne
+    @JoinColumn(name="venueID")
+    private Venue venue;
+
```

Comment:
look into a bidirectional relationship between Event and TicketCategory so you can return a list of TicketCategories in the response (as in the given samples).

Reply:
Done